doc/api/tracing.md

```diff
@@ -22,6 +22,8 @@ The available categories are:
 * `node.bootstrap`: Enables capture of Node.js bootstrap milestones.
 * `node.console`: Enables capture of `console.time()` and `console.count()`
   output.
+* `node.threadpool`: Enables capture of Node.js threadpool,
```

Comment:
We are capturing the event of works in the thread pool, not the thread pool itself, right?

```suggestion
* `node.threadpoolwork`: Enables capture of Node.js threadpool work,
```

Reply:
Done!